src/test/java/org/apache/commons/io/FileUtilsTest.java

```diff
@@ -1594,6 +1594,22 @@ public void testGetTempDirectoryPath() {
         assertEquals(System.getProperty("java.io.tmpdir"), FileUtils.getTempDirectoryPath());
     }
 
+    @Test
+    public void testGetTempDirectoryPathConsistentNoTrailingSeparator() {
+        String tempDirectoryPath = FileUtils.getTempDirectoryPathConsistent(false);
```

Comment:
This test is not valid: It does not assert the actual path is correct. For example, if the new API returns a "C:\Some\Odd\Dir" that is not the Java temp dir, then the test would pass.

Reply:
eh ok. I think that's overkill but I can change that. wait one and I'll push an update to the test

Reply:
done and done. I've added assertions to both tests that ensure the root path is the same as is reported by property `java.io.tmpdir`